.github/workflows/lighthouse-ci.yml

```diff
@@ -0,0 +1,45 @@
+name: Lighthouse CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    types: [labeled]
+
+jobs:
+  lhci:
+    name: Run Lighthouse CI
+    runs-on: ubuntu-latest
+
+    if: ${{ !contains(github.event.pull_request.labels.*.names, 'need-perf-check') }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
+
+      - name: Wait for the Netlify Preview
+        uses: jakepartusch/wait-for-netlify-action@v1
+        id: netlify
+        with:
+          site_name: 'wargabantuwarga'
+
+      - name: "Lighthouse CI assertion"
+        id: lhci
+        shell: bash
+        run: |
+          yarn global add @lhci/cli@0.8.x
+          echo -e "\n"
+          echo "Start collecting LH report..."
+          lhci collect --url=${{ steps.netlify.outputs.url }} -n=1
```

Comment:
Instead of using the deploy preview URL that is injected with some custom scripts by Netlify, can we get the original URL (the unique hash for each deployment instance)?

Please refer to [this comment ](https://github.com/kawalcovid19/wargabantuwarga.com/pull/19#issuecomment-877888775) for more context.

Reply:
Hmm. After looking at [the source code here](https://github.com/JakePartusch/wait-for-netlify-action/blob/d85a22dfa96b3f4537d8abdf2dab4ba2a6ac2079/index.js#L32), I don't think it's possible. 😅

Then let's go with this solution, for now, at least it can make the build on Netlify faster. 😁